internal/handlers/common/update.go

```diff
@@ -284,9 +286,8 @@ func processRenameFieldExpression(doc *types.Document, update *types.Document) (
 		doc.RemoveByPath(sourcePath)
 
 		// Set new path with old value
-		err = doc.SetByPath(targetPath, val)
-		if err != nil {
-			return changed, err
+		if err := doc.SetByPath(targetPath, val); err != nil {
+			return false, lazyerrors.Error(err)
```

Comment:
There is no test case for it, and I couldn't come up with one quickly, so I return `lazyerrors.Error` to make the fact we don't test it explicit.